strawberry/exceptions.py

```diff
@@ -176,3 +176,13 @@ def __init__(self, field_name: str, argument_name: str, argument_type: str):
         message = f'Argument "{argument_name}" on field "{field_name}" cannot be of type\
             "{argument_type}"'
         super().__init__(message)
+
+
+class InvalidSuperclassInterface(Exception):
+    def __init__(self, input_name: str, interface_names=List[str]):
+        interface_names = ", ".join(interface_names)
+        message = (
+            f"An Input class {input_name!r} cannot inherit "
+            f"from an Interface(s) {interface_names!r}"
+        )
```

Comment:
Why do these `fstring` expressions use `!r`? Neither a string or a list of strings should have a difference between `str()` and `repr()`.

Reply:
I put them so they are printed with ticks: `"InterfaceName"` `"['ClassOne', 'ClassTwo']"`

Reply:
Oh I guess there is a difference.